cpp/src/io/parquet/compact_protocol_writer.hpp

```diff
@@ -50,6 +50,7 @@ class CompactProtocolWriter {
   size_t write(const KeyValue&);
   size_t write(const ColumnChunk&);
   size_t write(const ColumnChunkMetaData&);
+  size_t write(const Statistics&);
```

Comment:
Please change all (this, above and below) to east const.

Reply:
updated to east const